src/cloud-api-adaptor/pkg/securecomms/ppssh/ppssh.go

```diff
@@ -0,0 +1,313 @@
+package ppssh
+
+import (
+	"bytes"
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"errors"
+	"fmt"
+	"net"
+	"os"
+	"sync"
+
+	"github.com/confidential-containers/cloud-api-adaptor/src/cloud-api-adaptor/pkg/securecomms/sshproxy"
+	"github.com/confidential-containers/cloud-api-adaptor/src/cloud-api-adaptor/pkg/securecomms/sshutil"
+	"golang.org/x/crypto/ssh"
+)
+
+const (
+	UNPROVEN_WN_PUBLIC_KEY_PATH = "/tmp/unprovenWnPublicKey"
+	SINGLETON_PATH              = "/tmp/sshSingleton"
```

Comment:
adding comment to indicate peer pod key and worker node key will be helpful

Reply:
added comment